esp/src/src-react/themes.ts

```diff
@@ -58,4 +61,84 @@ export const darkTheme: PartialTheme = {
         black: "#ffffff",
         white: "#000000",
     }
-};
\ No newline at end of file
+};
+
+const brandWeb: BrandVariants = {
+    10: "#061724",
```

Comment:
Where do these hexcodes come from, the older PartialThemes in themes.ts?

Reply:
I have added a README.md to this folder with some useful links.